sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/MainView.groovy

```diff
@@ -0,0 +1,81 @@
+package life.qbic.portal.sampletracking.components
+
+
+import com.vaadin.ui.HorizontalLayout
+import com.vaadin.ui.Label
+import com.vaadin.ui.VerticalLayout
+import com.vaadin.ui.themes.ValoTheme
+import life.qbic.portal.sampletracking.components.projectoverview.ProjectOverviewView
+import life.qbic.portal.sampletracking.components.sampleoverview.SampleOverviewView
+import life.qbic.portal.sampletracking.components.toggle.ToggleButton
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+class MainView extends VerticalLayout {
+
+    private final ProjectOverviewView projectOverviewView
+    private final SampleOverviewView sampleOverviewView
+    private final HorizontalLayout hotbar = new HorizontalLayout()
```

Comment:
if it has a hotbar, would it make sense to implement your interface here?

Reply:
As it does not need to implement the interface (it is not referenced anywhere else) I don't think it makes sense. If needed this can of course be changed in the future.